Sprint-3/2-mandatory-rewrite/1-get-angle-type.test.js

```diff
@@ -10,15 +10,27 @@ test("should identify right angle (90°)", () => {
 // Case 2: Identify Acute Angles:
 // When the angle is less than 90 degrees,
 // Then the function should return "Acute angle"
-
+test("should identify acute angle (45°)", () => {
+  expect(getAngleType(45)).toEqual("Acute angle");
+});
 // Case 3: Identify Obtuse Angles:
 // When the angle is greater than 90 degrees and less than 180 degrees,
 // Then the function should return "Obtuse angle"
+test("Should identify Obtuse angle (120)", () => {
+  expect(getAngleType(120)).toEqual("Obtuse angle");
+});
 
 // Case 4: Identify Straight Angles:
 // When the angle is exactly 180 degrees,
 // Then the function should return "Straight angle"
+test("Should identify Straight angle (180)", ()=>
+{
+  expect(getAngleType(180)).toEqual("Straight angle");  
+});
 
 // Case 5: Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+test("Should identify Reflex angle (320)", ()=>{
+  expect(getAngleType(320)).toEqual("Reflex angle");
+})
```

Comment:
you can also add a test for what if number is greater than 360, consider this as a challenge